mapentity/static/mapentity/mapentity.map.js

```diff
@@ -144,6 +144,10 @@ $(window).on('entity:map', function (e, data) {
         //this.options.filter.bboxfield.val(L.Util.getWKT(rect));
         $('#id_bbox').val(L.Util.getWKT(rect));
     });
+
+    // Disable scroll on overlay form to avoid map zoom change when scroll on layers list
+    layersControlDiv = document.getElementsByClassName('leaflet-control-layers-list')[0];
```

Comment:
You could add a `var` to your variable declaration.

Reply:
I used another selector eventually